tests/test_headers.py

```diff
@@ -207,14 +207,24 @@ def on_get(self, req, resp):
 
 class ContentLengthHeaderResource(object):
 
-    def __init__(self, content_length, body=None):
+    def __init__(self, content_length, body=None, data=None):
         self._content_length = content_length
         self._body = body
+        self._data = data
 
     def on_get(self, req, resp):
         # NOTE(kgriffs): Use stream_len for now to cover the deprecated alias
         resp.stream_len = self._content_length
-        resp.body = self._body
+
+        if self._body:
+            resp.body = self._body
+
+        if self._data:
+            resp.data = self._data
+
+    def on_head(self, req, resp):
+        # NOTE(kgriffs): Use stream_len for now to cover the deprecated alias
```

Comment:
Intent of this note is somewhat unclear (suspected copy-paste from L216?)

Reply:
oops, yeah. Copy-paste #fail - Let me fix that real quick

Reply:
fixed